scaleway/helpers_k8s.go

```diff
@@ -88,6 +89,57 @@ func waitK8SClusterDeleted(k8sAPI *k8s.API, region scw.Region, clusterID string)
 	return fmt.Errorf("Cluster %s has state %s, wants %s", clusterID, cluster.Status.String(), k8s.ClusterStatusDeleted.String())
 }
 
+func waitK8SPoolReady(k8sAPI *k8s.API, region scw.Region, poolID string) error {
+	pool, err := k8sAPI.WaitForPool(&k8s.WaitForPoolRequest{
+		PoolID:  poolID,
+		Region:  region,
+		Timeout: scw.DurationPtr(K8SPoolWaitForReadyTimeout),
+	})
+
+	if err != nil {
+		return err
+	}
+
+	if pool.Status == k8s.PoolStatusReady {
+		return nil
+	}
+	return fmt.Errorf("Pool %s has state %s, wants %s", poolID, pool.Status.String(), k8s.PoolStatusReady.String())
```

Comment:
nits:
- error should start with a lowercase letter.
- `.String()` methods are not required since `%s` will be looking for `fmt.Stringer` implementations.

Reply:
OK do you want me to fix similar lines in this file too ?

Reply:
Fixed in ```waitK8SClusterReady``` and ```waitK8SClusterDeleted``` too.